pkg/controllers/disruption/validation.go

```diff
@@ -148,12 +198,23 @@ func (v *Validation) ValidateCandidates(ctx context.Context, candidates ...*Cand
 }
 
 // ShouldDisrupt is a predicate used to filter candidates
-func (v *Validation) ShouldDisrupt(_ context.Context, c *Candidate) bool {
+func (v *validation) ShouldDisrupt(_ context.Context, c *Candidate) bool {
```

Comment:
Same comment here -- I'd try to stay DRY and just re-use the method's ShouldDisrupt